tellduslive.py

```diff
@@ -76,23 +76,54 @@ def __init__(self,
                  public_key,
                  private_key,
                  token,
-                 token_secret):
+                 token_secret,
+                 host=None):
+
         self._session = requests.Session()
-        self._session.auth = OAuth1(
-            public_key,
-            private_key,
-            token,
-            token_secret)
+        if host is not None:
+            self._token = token
+            self._session.headers = {"Authorization": "Bearer %s" % token}
+            self._api_url = "http://%s/api/" % host
+        else:
+            self._session.auth = OAuth1(public_key,
+                    private_key, token, token_secret)
+            self._api_url = API_URL
```

Comment:
I guess we could rename the API_URL constant later to TELLDUS_LIVE_API_URL or something.